src/qt/dash.cpp

```diff
@@ -483,7 +469,7 @@ void BitcoinApplication::initializeResult(bool success)
     if(success)
     {
         // Log this only after AppInitMain finishes, as then logging setup is guaranteed complete
-        qWarning() << "Platform customization:" << platformStyle->getName();
+        qWarning() << "Platform customization:" << gArgs.GetArg("-uiplatform", BitcoinGUI::DEFAULT_UIPLATFORM).c_str();
```

Comment:
This line, the `-uiplatform` option and `BitcoinGUI::DEFAULT_UIPLATFORM` const should be dropped.

Reply:
Why? Its still used for the platform specific css style thing we just introduced.

Reply:
Ahhh, right! nvm then :)